services/event/service/token_service.go

```diff
@@ -0,0 +1,15 @@
+package service
+
+import (
+	"net/http"
+
+	"github.com/HackIllinois/api/common/authtoken"
+	"github.com/HackIllinois/api/services/event/config"
+)
+
+func IsRequestFromStaffOrHigher(r *http.Request) bool {
+	token := r.Header.Get("Authorization")
+	is_at_least_staff, err := authtoken.IsAuthorized(config.TOKEN_SECRET, token, []authtoken.Role{authtoken.StaffRole, authtoken.AdminRole})
+
+	return err == nil && is_at_least_staff
+}
```

Comment:
This seems like a method that might want to be common, which would also make using the token secret much easier.

I don't see anything really event specific here.